src/main/controllers/CaseNameSearchResultsController.ts

```diff
@@ -18,4 +20,29 @@ export default class CaseNameSearchResultsController {
       res.render('error', req.i18n.getDataByLanguage(req.lng).error);
     }
   }
+
+  public async post(req: PipRequest, res: Response): Promise<void> {
```

Comment:
why has this been added? caseNameSearchResultsController used to post data to pending-subscriptions page